file.go

```diff
@@ -118,8 +119,8 @@ func (fi *File) Open(flags int, sync bool) (FileDescriptor, error) {
 // pretty much the same thing as here, we should at least call
 // that function and wrap the `ErrNotUnixfs` with an MFS text.
 func (fi *File) Size() (int64, error) {
-	fi.nodelk.Lock()
-	defer fi.nodelk.Unlock()
```

Comment:
Doesn't have to happen in this PR but, ideally, we'd call `GetNode()` here instead of holding the lock.

Reply:
Yes, I have a couple of `TODO`s to refactor that (including also adding a `SetNode()` method).